src/studio/src/designer/frontend/packages/schema-editor/src/features/editor/schemaEditorSlice.ts

```diff
@@ -319,9 +323,14 @@ const schemaEditorSlice = createSlice({
       const {
         id, focusName,
       } = action.payload;
-      state.selectedId = id;
-      state.selectedTreeNodeId = getDomFriendlyID(id);
       state.focusNameField = focusName;
+      if (state.selectedEditorTab === '1') {
```

Comment:
could we replace the tab-ids ('0' and '1') with 'properties' and 'definitions', respectfully?
I might be missing something, but I can't really see why we need to explicitly have numbered ids here.

Reply:
Great suggestiong. Fixed.